src/sentry/web/api.py

```diff
@@ -486,6 +486,7 @@ def process(self, request, project, key, auth, helper, data, **kwargs):
             helper.ensure_does_not_have_ip(data)
 
         # mutates data (strips a lot of context if not queued)
+        data['key'] = key.id
```

Comment:
I'd like to have a larger discussion about this. It has come up before that we should track which ProjectKey created an event.

See: https://github.com/getsentry/sentry/issues/2709

So if we do this, I think we should do it with this in mind and plan on a longer term way to surface this in the product.

Mentioning this because I don't think we should be temporarily putting this data into the event blob just to pop it out later simply for this. This is valuable data on an event that we can use, so I think we should keep it.

Reply:
ok, so is your suggestion for now to put it in the event blob and not pop it out?